src/stores/useDashboard.ts

```diff
@@ -260,7 +260,7 @@ export const useDashboard = defineStore('dashboard', {
   state: () => {
     const favMap = JSON.parse(
       localStorage.getItem('favoriteMap') ??
-        '{"cosmos":true, "osmosis":true, "nibiru":true, "nibiru-itn-3":true}'
+        '{"cosmos":true,"osmosis":true,"nibiru":true,"nibiru-itn-3":true,"cataclysm-1":true,"nibiru-testnet-1":true}'
```

Comment:
Maybe in a future PR, we can get rid of Cosmos and Osmosis. We have no business hosting their explorers on our webpage.